src/components/Chips/Chips.tsx

```diff
@@ -93,21 +99,23 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
     const hasCloseButton = !readOnly && !disabled;
 
     const [hoverRef, isHovered] = useHover();
+    const [focusRef, isFocused] = useFocus();
```

Comment:
Sorry for being evil but because our standard right now is no "is" prefix for new props for alignment, let's convert isClickable to be "backward compatibility " and add clickable prop :(

Reply:
😈